packages/ia-components/sandbox/audio-players/youtube-player.jsx

```diff
@@ -56,6 +56,7 @@ class YoutubePlayer extends Component {
     const { id } = this.state;
     const trackChanged = id !== nextProps.id;
     if (trackChanged) {
+      clearTimeout(window.timer);
```

Comment:
Can we reference the timer on the component instead of the window? Then we're better safeguarded from some other code overwriting `window.timer`

Reply:
Done!